07.ls_object/short_format_list.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative 'file_list'
+require_relative 'fetch_file_detail'
+
+COLUMNS = 3
+SPACE_FOR_COLUMNS = 2
+
+class ShortFormatList
+  include FetchFileDetail
+
+  private
+
+  def fetch_file_details(path, reverse_required, hidden_file_required)
+    file_list = FileList.new(path, reverse_required, hidden_file_required)
+    adjust_list_for_display(file_list.name_list)
+  end
+
+  def adjust_list_for_display(files)
+    rows = (files.size.to_f / COLUMNS).ceil
+    lines = []
+    max_file_names = []
+    files.each_with_index do |file_name, i|
+      current_row = i % rows
+      current_column = i / rows
+      lines[current_row] = [] if current_column.zero?
+      max_file_names[current_column] ||= 0
+
+      lines[current_row] << file_name
+      max_file_names[current_column] = calc_file_name_size(file_name) if max_file_names[current_column] < file_name.size
```

Comment:
```suggestion
      max_file_names[current_column] = calc_file_name_size(file_name) if max_file_names[current_column] < calc_file_name_size(file_name.size)
```

にしないと、最大値の判定を間違ってしまうのでは？と思いました。

Reply:
判定間違えてました。
`calc_file_name_size(file_name)`に修正します。